packages/dx-react-grid-demos/src/demo-sources/grid-editing/cell-editing-tab-with-edit-column.jsxt

```diff
@@ -16,6 +15,13 @@ import {
 
 const getRowId = row => row.id;
 
+const FocusableCell = (props) => {
+  const { onClick } = props;
+  return (
+    <Table.Cell {...props} tabIndex={0} onFocus={onClick} />
+  );
+};
+
```

Comment:
```jsx
const FocusableCell = ({ onClick, ...restProps }) => (
  <Table.Cell {...restProps} tabIndex={0} onFocus={onClick} onClick={onClick} />
);
```

Should we add this custom component to all inline cell editing demos?

Reply:
Because we decided that cell editing without tabulation is not complete feature. We should display this workaround anywhere where we are using this feature.

Reply:
We decided that at least one demo (basic) should show plugin's possibilities without custom components. But I agree that other demos with inline cell editing should have this component.